src/ucp/wireup/wireup_cm.c

```diff
@@ -858,7 +878,7 @@ ucs_status_t ucp_ep_client_cm_create_uct_ep(ucp_ep_h ucp_ep)
                                 UCT_EP_PARAM_FIELD_USER_DATA                  |
                                 UCT_EP_PARAM_FIELD_SOCKADDR                   |
                                 UCT_EP_PARAM_FIELD_SOCKADDR_CB_FLAGS          |
-                                UCT_EP_PARAM_FIELD_SOCKADDR_PACK_CB           |
+                                UCT_EP_PARAM_FIELD_CM_RESOLVE_CB              |
```

Comment:
if UCT_EP_PARAM_FIELD_SOCKADDR_PACK_CB isn't used, why do we check for it in uct? you plan to remove it as next step?

also, UCT_EP_PARAM_FIELD_CM_RESOLVE_CB is used here but isn't checked for in uct, why?

Reply:
API has to be compatible, so we will support both from now

Reply:
yes but is set here UCT_EP_PARAM_FIELD_CM_RESOLVE_CB but uct doesn't check for it..

Reply:
do you mean like this?
https://github.com/openucx/ucx/blob/2823e3aa7a023de5128c5cb829ee9e4e566cc6b3/src/uct/ib/rdmacm/rdmacm_cm.c#L159

Reply:
or this?
https://github.com/openucx/ucx/blob/2823e3aa7a023de5128c5cb829ee9e4e566cc6b3/src/uct/base/uct_cm.c#L172

Reply:
we usually check the flag before accessing its relevant field. like here - https://github.com/openucx/ucx/blob/master/src/uct/ib/rdmacm/rdmacm_cm_ep.c#L348
https://github.com/openucx/ucx/blob/master/src/uct/ib/rdmacm/rdmacm_cm_ep.c#L345

Reply:
see `UCT_CM_SET_CB`